curriculum/challenges/english/01-responsive-web-design/applied-visual-design/learn-about-tertiary-colors.english.md

```diff
@@ -11,12 +11,12 @@ Computer monitors and device screens create different colors by combining amount
 Tertiary colors are the result of combining a primary color with one of its secondary color neighbors. For example, within the RGB color model, red (primary) and yellow (secondary) make orange (tertiary). This adds six more colors to a simple color wheel for a total of twelve.
 There are various methods of selecting different colors that result in a harmonious combination in design. One example that can use tertiary colors is called the split-complementary color scheme. This scheme starts with a base color, then pairs it with the two colors that are adjacent to its complement. The three colors provide strong visual contrast in a design, but are more subtle than using two complementary colors.
 Here are three colors created using the split-complement scheme:
-<table class="table table-striped"><thead><tr><th>Color</th><th>Hex Code</th></tr><thead><tbody><tr><td>orange</td><td>#FF7D00</td></tr><tr><td>cyan</td><td>#00FFFF</td></tr><tr><td>raspberry</td><td>#FF007D</td></tr></tbody></table>
+<table class="table table-striped"><thead><tr><th>Color</th><th>Hex Code</th></tr><thead><tbody><tr><td>orange</td><td>#FF7F00</td></tr><tr><td>cyan</td><td>#00FFFF</td></tr><tr><td>raspberry</td><td>#FF007F</td></tr></tbody></table>
 </section>
 
 ## Instructions
 <section id='instructions'>
-Change the <code>background-color</code> property of the <code>orange</code>, <code>cyan</code>, and <code>raspberry</code> classes to their respective colors. Make sure to use the hex codes as orange and raspberry are not browser-recognized color names.
+Change the <code>background-color</code> property of the <code>orange</code>, <code>cyan</code>, and <code>raspberry</code> classes to their respective colors. Make sure to use the hex codes and not the color names. While <code>orange</code> is a color name recognized by most browsers, the shade is not exactly the same as the tertiary orange color (#FF7F00).
```

Comment:
@beaucarnes 

If a 4th test like below was added:
```yml
  - text: All <code>background-color</code> values for the <code>div</code> elements' classes should be hex codes and not color names.
    testString: assert(!/background-color:\s(orange|cyan|raspberry)/.test(code), 'All <code>background-color</code> values for the <code>div</code> elements' classes should be hex codes and not color names.');  
```
then line 19 would not have to have that last sentence.
```suggestion
Change the <code>background-color</code> property of the <code>orange</code>, <code>cyan</code>, and <code>raspberry</code> classes to their respective colors. Make sure to use the hex codes and not the color names.
```
Otherwise, the 2nd test would still pass if a user used `cyan` for the the `cyan` class definition.   Plus, the instructions remain concise and the test text reinforces what the instructions already have stated:
> Make sure to use the hex codes and not the color names.

What do you think?